ui/vite.config.js

```diff
@@ -14,8 +15,9 @@ let build_config =
 export default defineConfig({
   server: {
     port: 3002,
-    //https: true,
+    https: true,
```

Comment:
The readme should probably be updated :-)

Reply:
done